releases/release-6.5.12.md

```diff
@@ -0,0 +1,187 @@
+---
+title: TiDB 6.5.12 Release Notes
+summary: Learn about the improvements and bug fixes in TiDB 6.5.12.
+---
+
+# TiDB 6.5.12 Release Notes
+
+Release date: February 27, 2025
+
+TiDB version: 6.5.12
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.5/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v6.5/production-deployment-using-tiup)
+
+## Compatibility changes
+
+- Set a default limit of 2048 for DDL historical tasks retrieved through the [TiDB HTTP API](https://github.com/pingcap/tidb/blob/release-7.5/docs/tidb_http_api.md) to prevent OOM issues caused by excessive historical tasks [#55711](https://github.com/pingcap/tidb/issues/55711) @[joccau](https://github.com/joccau)
```

Comment:
```suggestion
- Set a default limit of 2048 for DDL historical tasks retrieved through the [TiDB HTTP API](https://github.com/pingcap/tidb/blob/release-6.5/docs/tidb_http_api.md) to prevent OOM issues caused by excessive historical tasks [#55711](https://github.com/pingcap/tidb/issues/55711) @[joccau](https://github.com/joccau)
```